docs/index.md

```diff
@@ -144,37 +144,48 @@ npm run kakunin
 In order to make cucumber steps autosuggestion work properly in JetBrains tools, make sure your project is `ECMAScript 6` compatible and you have `cucumberjs` plugin installed.
 Due to non-resolved issue in Jetbrains editors ([see here](https://youtrack.jetbrains.com/issue/WEB-11505)) we'll have to do one more step:
  
-Go to `step_definitions` directory 
+Go to `step_definitions` directory
 ```bash
 cd step_definitions
+mkdir common && mkdir rest && mkdir web
```

Comment:
I checked it, and Kakunin sees the steps in the `step_definitions/web/step.js` directory. Also, the autosuggestion works. @sethii 
So keeping it in separated folders might be better.